site/_posts/2018-12-05-gandiva-donation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Gandiva: A LLVM-based Analytical Expression Compiler for Apache Arrow"
+date: "2018-12-05 00:00:00 -0400"
+author: wesm
```

Comment:
This should be jacques-n here. Will change

Reply:
Yes, that makes sense. Thanks, Wes!